tests/e2e/csi_static_provisioning_basic.go

```diff
@@ -1755,7 +1815,8 @@ var _ = ginkgo.Describe("Basic Static Provisioning", func() {
 		framework.Logf("Datastore type: %s", dataStoreType)
 
 		if dataStoreType != "vsan" {
-			ginkgo.Skip("Skipping static provisioning - import VMDK test Since the testbed dont have vSAN datastore - Because for this test uses vSAN default datastore policy ")
+			ginkgo.Skip("Skipping static provisioning - import VMDK test Since the testbed dont have vSAN datastore - " +
```

Comment:
I think there should be no space before "+".  Have you run "go fmt"?

Reply:
Yes, this is done by "go fmt".

"Go fmt" is inconsistent while deciding the white spaces.

"Concatenating strings in an assignment requires spaces around the +, same for inside function calls that have just 1 argument, but when there's a , inside the function call, the + must not have spaces..."
 -- https://github.com/golang/go/issues/34426
 
 Golang is still a young language, compared to C and C++, IMHO.

Reply:
Yeah, it is really inconsistent:).